test

```diff
@@ -10,9 +10,10 @@ else
     TARGET="fades tests"
 fi
 
+FADES='./bin/fades -r requirements.txt'
 
-nosetests3 -v -s $TARGET
-flake8 $TARGET --max-line-length=99 --select=E,W,F,C,N
+$FADES -x nosetests -- -v -s $TARGET
+$FADES -x flake8 -- $TARGET --max-line-length=99 --select=E,W,F,C,N
 
 # On Archlinux rst2html is rst2html2
 if hash rst2html > /dev/null 2>&1; then
```

Comment:
This magic of "how rst2html is named in Arch" is no longer needed, as now it's used from the virtualenv